docs/standard/net-standard.md

```diff
@@ -87,11 +73,11 @@ Derivative artifacts are provided to enable more convenient reading and to enabl
 
 ### Package representation
 
-The primary distribution vehicle for the .NET Standard reference assemblies is [NuGet packages](../core/packages.md). Implementations will be delivered in a variety of ways, appropriate for each .NET implementation.
+The primary distribution vehicle for the .NET Standard reference assemblies is [NuGet packages](../core/packages.md). Implementations are delivered in a variety of ways, appropriate for each .NET implementation.
 
-NuGet packages target one or more [frameworks](frameworks.md). The .NET Standard packages target the ".NET Standard" framework. You can target the .NET Standard Framework using the `netstandard` [compact TFM](frameworks.md) (for example, `netstandard1.4`). Libraries that are intended to run on multiple runtimes should target this framework. 
+NuGet packages target one or more [frameworks](frameworks.md). The .NET Standard packages target the ".NET Standard" framework. You can target the .NET Standard Framework using the `netstandard` [compact TFM](frameworks.md) (for example, `netstandard1.4`). Libraries that are intended to run on multiple runtimes should target this framework. If you want a broader set of APIs, you should target `netstandard2.0` since the number of available APIs more than doubled between .NET Standard 1.6 and 2.0.
```

Comment:
- nit: should framework in .NET Standard Framework be capitalized (I suspect not)
- The "if you want" sentence seems out of place. I think a slight rewrite helps: "You can target the .NET Standard framework using the \`netstandard\` \[compact target framework moniker](link) (TFM). Libraries that are intended to run on multiple platforms should target at least \`netstandard1.4\`. For the broadest set of APIs, target...